cmd/dra-example-controller/driver.go

```diff
@@ -70,7 +70,11 @@ func (d driver) GetClassParameters(ctx context.Context, class *resourcev1.Resour
 	return &dc.Spec, nil
 }
 
-func (d driver) GetClaimParameters(ctx context.Context, claim *resourcev1.ResourceClaim, class *resourcev1.ResourceClass, classParameters interface{}) (interface{}, error) {
+func (d driver) GetClaimParameters(
+	ctx context.Context,
+	claim *resourcev1.ResourceClaim,
+	class *resourcev1.ResourceClass,
+	classParameters interface{}) (interface{}, error) {
```

Comment:
I'm not a fan of wrapping method signatures like this. In any case, this is not relevant to this PR, so it shouldn't be included here. We can have a separate discussion about whether we want to do this or not and then open a PR to address it across the code base.